### PR TITLE
Move ffmpeg-installer to optional packages

### PR DESCRIPTION
**Pull request by benschwarz**
- **What kind of change does this PR introduce?**

Security improvement

- **What is the current behavior?** (You can also link to an open issue here)

ffmpeg-installer is installed as a direct dependency of this library

- **What is the new behavior (if this is a feature change)?**

ffmpeg-installer is moved to `optionalDependencies` from `dependencies`

- **Other information**:

ffmpeg installer has optional dependencies for each cpu architecture, each ships [with `postinstall` scripts that could be used as an attack vector](https://socket.dev/npm/package/@ffmpeg-installer/ffmpeg)

By moving ffmpeg-installer to an optional dependency, we can hopefully mitigate this effect. [This also appears to match the intent of how the library is used. ](https://github.com/prasanaworld/puppeteer-screen-recorder/blob/2241c5b4aac3a3a9c35706edb07465a6404be8c7/src/lib/pageVideoStreamWriter.ts#L67-L77)

If there's something I can do to help improve this PR or land this change, please let me know. Happy to help where I can!

### CODE REVIEW

**Comment by prasanaworld** [comment]
Hi @benschwarz,

Thanks for pointing out the security risk and PR. I appreciate it. 

It would be great, if you could update your PR against latest changes, Please.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppeteer-screen-recorder",
-  "version": "2.0.2",
+  "version": "2.0.3",
```
Comment (prasanaworld) [inline package.json:3]: Version updated is not required to be done manually. This repo is configured to used `standard-version` so the version will updated before the release automatically by the build tool. 

Could you please undo the changes.

Comment (benschwarz) [reply package.json:3]: No problem. Done.

**Comment by benschwarz** [comment]
@prasanaworld I've rolled back the version bump and merged the updates from `main`. Thanks!

**Comment by benschwarz** [comment]
@prasanaworld Is there anything else you'd like to see on this PR? LMK. I'm keen to have this released!  🚀

**Review by prasanaworld — APPROVED** [review]

**Comment by prasanaworld** [comment]
Looks good to me. Thanks for your contribution

**Comment by benschwarz** [comment]
Thanks @prasanaworld. As far as I can see, the build tool hasn't automatically released an updated npm package. Could you give it a nudge?

**Comment by benschwarz** [comment]
Thanks @prasanaworld. As far as I can see, the build tool hasn't automatically released an updated npm package. Could you give it a nudge?
